src/main/res/values/strings.xml

```diff
@@ -1267,13 +1267,13 @@
 
 	<!-- 2020-08-18 -->
 	<string name="pref_behaviour_video_zoom_default_key" translatable="false">pref_behaviour_video_zoom_default</string>
-	<string name="pref_behaviour_video_zoom_default_title">Zoom into videos by default</string>
+	<string name="pref_behaviour_video_zoom_default_title">Zoom in on videos by default</string>
 
 	<!-- 2020-08-23 -->
 	<string name="image_gallery_removed_title">Gallery removed</string>
-	<string name="image_gallery_removed_message">This gallery has been removed and is no longer accessible.</string>
+	<string name="image_gallery_removed_message">This removed gallery is no longer accessible.</string>
```

Comment:
Original made more sense

Reply:
Why? When did it get removed?

Reply:
It's deeply unidiomatic and relegates the most essential information ("gallery has been removed") to a lesser position, implying that it's something the user should already been aware of. Let's keep the original please.